spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/CloudEurekaTransportConfig.java

```diff
@@ -18,12 +18,14 @@
 
 import com.netflix.discovery.shared.transport.EurekaTransportConfig;
 
-import lombok.Data;
+import org.apache.commons.lang.builder.EqualsBuilder;
```

Comment:
Probably not (unless it was in a collection), but it would be a breaking change to remove them since they were there.